integration-webhook/src/test/java/org/symphonyoss/integration/webhook/WebHookIntegrationTest.java

```diff
@@ -108,6 +108,20 @@
 
   private static final String INTEGRATION_USER = "jiraWebHookIntegration";
 
+  private static final String XML_WITH_PROLOG =
```

Comment:
Although I know this is not an elegant solution, writing an invalid XML file is not either. But I can change it if you want.